BrowserKit/Sources/ToolbarKit/ToolbarButton.swift

```diff
@@ -74,7 +75,27 @@ class ToolbarButton: UIButton, ThemeApplicable {
         configuration = updatedConfiguration
     }
 
-    // MARK: ThemeApplicable
+    private func configureLongPressGestureRecognizerIfNeeded(for element: ToolbarElement) {
+        if element.hasLongPressGestureRecognizer {
+            let longPressRecognizer = UILongPressGestureRecognizer(
+                target: self,
+                action: #selector(handleLongPress)
+            )
+            addGestureRecognizer(longPressRecognizer)
```

Comment:
`configure(element:)` could potentially be called multiple times. So to make sure that everything is set up correctly we should remove any existing gesture recognizers.

Reply:
I think it is faster to check if we already have a long press gesture recognizer instead of removing and re adding it.